scripts/start.py

```diff
@@ -178,6 +184,14 @@ def main(args: Optional[Sequence[str]] = None) -> None:
             skip_sdk_update_check=True,
             port=PORT_NUMBER_FOR_GAE_SERVER))
 
+        if parsed_args.contributor_dashboard_debug:
```

Comment:
/cc @IamEzio since tests for this will need to be added in #15919 if this PR merges first